### PR TITLE
fix LDPATH

### PR DESCRIPTION
**Pull request by v-p-b**
fixes https://github.com/stef/libopaque/issues/35

### CODE REVIEW

File: src/makefile [hunk 1]
```diff
@@ -23,7 +23,7 @@ endif
 
 ifdef OPRFHOME
 	OPRFINCDIR=$(OPRFHOME)
-	LDFLAGS+= -Wl,rpath,$(OPRFHOME)
+	LDFLAGS+= -L$(OPRFHOME)
```
Comment (stef) [inline src/makefile:26]: this we tried already in https://github.com/stef/libopaque/commit/65b15eaf4114f75c2d55b54cdbb6de85a314fd5e

File: src/makefile [hunk 2]
```diff
@@ -84,7 +84,7 @@ test: tests
 	LD_LIBRARY_PATH=. ./tests/opaque-munit$(EXT) --fatal-failures
 
 utils/opaque: utils/main.c
-	gcc $(CFLAGS) -I. -o utils/opaque utils/main.c -L. -lopaque -lsodium
+	gcc $(CFLAGS) -I. -o utils/opaque utils/main.c -L. $(LDFLAGS) -lopaque -lsodium
```
Comment (stef) [inline src/makefile:87]: well, this is new....

File: src/makefile [hunk 3]
```diff
@@ -84,7 +84,7 @@ test: tests
...
```
Comment (stef) [inline src/makefile:88]: i think i nailed it now with https://github.com/stef/libopaque/commit/e258d6a58f7dca3e8fc20aebe49f97bbd6373e95

File: src/makefile (hunk 1 above)
Comment (v-p-b) [reply src/makefile:26]: maybe I tested incorrectly (no make clean?)? Can't recall, but now it works.

**Comment by stef** [comment]
merged. thanks!

**Pull request merged**